packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -113,16 +141,17 @@ export class StatusBarImpl extends VirtualWidget implements StatusBar {
                 rightEntries.push(this.renderElement(entry));
             }
         });
-        const leftElements = h.div({ className: 'area left' }, VirtualRenderer.flatten(leftEntries));
-        const rightElements = h.div({ className: 'area right' }, VirtualRenderer.flatten(rightEntries));
-        return VirtualRenderer.flatten([leftElements, rightElements]);
+        const leftElements = <div className="area left"><React.Fragment>{leftEntries}</React.Fragment></div>;
+        const rightElements = <div className="area right"><React.Fragment>{rightEntries}</React.Fragment></div>;
+
+        return [leftElements, rightElements];
     }
 
     protected createAttributes(entry: StatusBarEntry): StatusBarEntryAttributes {
         const attrs: StatusBarEntryAttributes = {};
 
         if (entry.command) {
-            attrs.onclick = () => {
+            attrs.onClick = () => {
```

Comment:
we should get rid of it: https://stackoverflow.com/questions/36677733/why-shouldnt-jsx-props-use-arrow-functions-or-bind